content/guides/secure-payments-by-bookingsync.md

```diff
@@ -90,11 +93,58 @@ Here's an example of the expected payload:
 ~~~
 
 
-There are 2 possible responses:
+There are 3 possible responses:
 
 1. Successful - with status code `200` and no response body - it means that the payment has been successfully authorized and has been enqueued to be captured asynchronously. Once the credit card payment is captured, it will be confirmed on the BookingSync side - the payment will be updated with `transaction_id` and `paid_at` values and the related booking will be confirmed - marked as booked.
 
-2. Failure - with status code `422` and validation errors inside response body. Here's an example of possible validation errors:
+2. Partial Success - with status code `202` and response body dependent on Payment Gateway. This status means that payment requires authentication.
+
+BookingPay Gateway
+
+~~~json
+{
+  "enrollmentId": "sample_enrollment_id_123",
+  "acsURL": "https://test-threedsecure.centralpay.net/acs",
+  "paReq": "paReq",
+  "gateway_name": "booking_pay",
+  "uuid": "lapw8u3k-9583-38l3-m5d9-51c0af5df8cb",
+  "order_id": "BS-123"
+}
+~~~
+
+Ogone Gateway
+
+~~~json
+{
+  "gateway_name": "ogone",
+  "HTML_ANSWER": "html_answer_that_is_encoded_form",
```

Comment:
I think it would be good to mention that it is Base64 encoded, otherwise it's asking for questions on API support :P